src/gui/settings/foundServers.html

```diff
@@ -0,0 +1,337 @@
+            background-color: rgba(0,0,0,0.2);
+        }
+        .buttonGreyedOut {
+            border: 2px solid rgba(0,0,0,0.2);
+            color: rgba(0,0,0,0.2);
+        }
+        
+        .rowName {
+            font-weight: 600;
+        }
+        
+        .objectNameInput {
+            display: inline-block;
+            margin-top: 20px;
+            width: 50% !important;
+        }
+        
+        .addObjectButton {
+            display: inline-block;
+        }
+        
+    </style>
+</head>
+<body style="background-color: #efeff4;">
+<div style="position: absolute; right: 0; width: 65px; top:0; bottom: 0; background-color: #222222"></div>
+<div>
+
+    <header class="bar bar-nav" style="right:65px; left:0">
+        <a class="" href="index.html" data-ignore="push">
+            <button class="btn btn-link btn-nav pull-left" style="user-select: none; -webkit-user-select: none">
+                <span class="icon icon-left-nav"></span>
+                Back
+            </button></a>
+        <div id="foundServersTitle" class="title">
+            <div class="flexItemWide">Name</div>
+            <div class="flexItem">Version</div>
+            <div class="flexItem">Address</div>
+        </div>
+    </header>
+
+    <!-- Wrap all non-bar HTML in the .content div (this is actually what scrolls) -->
+    <div class="content" style="right:65px;">
+        <div class="card">
+            <ul class="table-view" id="serverList"></ul>
+        </div>
+    </div>
+
+    <template id="tableEntry">
+            <li class="table-view-cell table-view-cell"></li>
+    </template>
+
+    <!-- Template for a single row that either contains server info -->
+    <template id="row">
+        <div>
+            <div class="infoRow serverInfoRow" style="height: 30px; font-size: large">
+                <img class="flexItem media-object pull-left" src="../../../svg/discovery.svg" style="width: 26px;
+                height: 26px; margin-top: -2px; margin-bottom: -5px; pointer-events: none">
+                <div class="flexItemWide rowName" style="width: 130px; overflow: hidden;"></div>
+                <div style="width: 70px; text-align: center">
+                    <span class="badge rowVersion"></span>
+                </div>
+                <div class="flexItem" style="width: 140px; text-align: center">
+                    <span class="badge rowIP"></span>
+                </div>
+            </div>
+        </div>
+    </template>
+
+    <!-- Template for a row of buttons that can be added underneath a server row by tapping on it -->
+    <template id="buttonRow">
+        <div class="buttons">
+            <div class="infoRow" style="height: 40px; font-size: large; justify-content: start; align-items: center; text-align: center">
+                <div style="width: 100%;">
+                    <form class="addObjectForm">
+                      <input type="text" name="name" pattern="[A-Za-z]+" class="objectNameInput"/>
```

Comment:
Nice attention to detail with the pattern field to only allow valid object names